### DISCUSSION TITLE
Query engine library for current platform could not be found

### DISCUSSION THREAD

**Discussion by charklewis** [prisma/prisma#15500] [Q&A]
I am trying to get prisma working in a dockerised environment and continue to come across the below error. I am on a MBP with M1 with my docker image `node:16`. 

```
Query engine library for current platform \"linux-arm64-openssl-1.1.x\" could not be found.\nYou incorrectly pinned it to linux-arm64-openssl-1.1.x\n\nThis probably happens, because you built Prisma Client on a different platform.
```

How can I resolve this? Is there a specific version I should be pinning?

**Comment by janpio** [comment]
What version of Prisma are you trying to use?

**Reply by charklewis** [reply]
I am using `4.3.1`. I am not pinning any specific library and I think there is confusion somewhere between it running in docker and the version installed on my M1.

**Reply by pantharshit00** [reply]
This might be related to https://github.com/prisma/prisma/issues/8478.

Can you please share more details like your schema and docker image? If you can provide a small reproduction in a Git repo that would be even more helpful.

**Reply by vladi-stevanovic** [reply]
👋 Hello @charklewis did you perhaps have a chance to check Harshit's question? 
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
